deps/rabbit/src/rabbit_khepri.erl

```diff
@@ -414,7 +414,13 @@ post_add_member(JoiningNode, JoinedNode, Error) ->
 %% @private
 
 leave_cluster(Node) ->
-    retry_khepri_op(fun() -> remove_member(Node) end, 60).
+    case retry_khepri_op(fun() -> remove_member(Node) end, 60) of
+        ok ->
+            rabbit_event:notify(node_deleted, [{node, Node}]),
```

Comment:
Yes, it is emitted from `rabbit_mnesia`. It should be be okay to emit it from above, but I thought that better not to touch Mnesia as it is on its way to deprecation

Reply:
The idea is mostly to not put something that isn't Khepri-specific in `rabbit_khepri`. If the event was not emitted from `rabbit_mnesia` in the first place, there would have been no bug to fix (except the consistency one which is separate).

Reply:
I like your change, thank you!

One question: at the time `notify_left_cluster/1` is called in `rabbit_db_cluster:join/2`, with Khepri, the node is still part of the cluster. There is no good solution because the reset is handled internally by `khepri_cluster:add_member/1`. Do you think there is any harm by emitted that event before the actual leave?

Reply:
I'm just investigating that. The failing test that @michaelklishin reported fails just with this last change.

Reply:
The reason that testcase is failing, is because `forget_cluster_node` cli command skips the `rabbit_db_cluster` layer on the fallback for mixed-version clusters. Older versions, don't thave that module